go.sum

```diff
@@ -1,36 +1,97 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Can you run `go mod tidy` and `go mod verify` please?

Reply:
```
 REDACTED  MCW0CDP3YY  ~  go  …  github.com  hashicorp  vault-plugin-auth-gcp   instance-jwt-auth  $    go mod tidy
 REDACTED  MCW0CDP3YY  ~  go  …  github.com  hashicorp  vault-plugin-auth-gcp   instance-jwt-auth  $    go mod verify
all modules verified
 REDACTED  MCW0CDP3YY  ~  go  …  github.com  hashicorp  vault-plugin-auth-gcp   instance-jwt-auth  $    git diff
 REDACTED  MCW0CDP3YY  ~  go  …  github.com  hashicorp  vault-plugin-auth-gcp   instance-jwt-auth  $    
```

Didn't generate a diff.